src/style.css

```diff
@@ -0,0 +1,101 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+img {
+  position: absolute;
+  top: 46px;
+  left: 35px;
+}
+
+.imgSmall {
+  width: 11px;
+  top: 126px;
+  left: 22px;
+}
+
+.search-input-big {
+  display: flex;
+  width: 100%;
+  height: 70px;
+  font-family: inherit;
+  font-size: 16px;
+  font-weight: 300;
+  line-height: 22px;
+  margin-top: 20px;
+  padding-left: 62px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
```

Comment:
Check the order of properties everywhere, please. You can use this [link](https://9elements.com/css-rule-order/)